openfisca_france/parameters/prestations_sociales/prestations_familiales/prestations_generales/cf/cf_plaf/plafond_ressources_majoration/biactifs_isoles.yaml

```diff
@@ -17,7 +17,11 @@ values:
   2021-01-01:
     value: 4367
   2022-01-01:
-    value: 4376
+    value: 4380
+  2023-01-01:
+    value: 4450
+  2024-01-01:
+    value: 4680
```

Comment:
Je ne trouve pas les valeurs de 2023 et 2024 dans les références mises ci dessous. De plus je ne comprends pas la correction pour 2022, la valeur "4376" est bien dans la référence correspondante.

Reply:
Ah noter que ce paramètre n'est appelé dans aucune formule d'openfisca france

Reply:
Bonjour, ce montant correspond à la moitié du plafond de base. Ce montant est aussi disponible à la page 76 du BO : https://sante.gouv.fr/fichiers/bo/2023/2023.24.sante.pdf#page=76
Je ne sais pas pourquoi je l'ai arrondi à la 10aine d'euro près par contre (ça doit venir de notre fichier excel de suivi de barême). On peut fix l'arrondi + ajouter le BO aux docs si c'est bon pour OF le BO ?

Reply:
> Bonjour, ce montant correspond à la moitié du plafond de base. Ce montant est aussi disponible à la page 76 du BO : https://sante.gouv.fr/fichiers/bo/2023/2023.24.sante.pdf#page=76 Je ne sais pas pourquoi je l'ai arrondi à la 10aine d'euro près par contre (ça doit venir de notre fichier excel de suivi de barême). On peut fix l'arrondi + ajouter le BO aux docs si c'est bon pour OF le BO ?